src/layouts/_proxied-dot-io/vault/index.tsx

```diff
@@ -11,11 +11,14 @@ import Footer from 'components/_proxied-dot-io/vault/footer-with-props'
 import ProductSubnav from 'components/_proxied-dot-io/vault/subnav'
 import productData from 'data/vault.json'
 import query from './query.graphql'
+import { trackGoal as trackFathomGoal } from 'fathom-client'
```

Comment:
@BRKalow did we ever make a generic analytics client? I feel like we talked about it, but I can't seem to track down if we ever implemented it.

Reply:
https://app.asana.com/0/search/1202008050032049/1201931300013818

Reply:
okay great so i'm not just imagining that!